index.html

```diff
@@ -10,6 +10,7 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
+  <div class="tech-div">
```

Comment:
Instead of using a name like `<something>-<tag>`, use a better naming that is a little more self explanatory, like `techs-container`